opentelemetry-sdk/src/trace/span_processor.rs

```diff
@@ -354,7 +369,10 @@ impl<R: RuntimeChannel> BatchSpanProcessorInternal<R> {
                     let export_task = self.export();
                     let task = async move {
                         if let Err(err) = export_task.await {
-                            global::handle_error(err);
+                            otel_error!(
+                                name: "BatchSpanProcessor.OnEnd.Error",
```

Comment:
this doesn't look correct to me, as this is occurring in background thread/task, and *not* in BSP's OnEnd method.

Reply:
Good point. Changed it to `BatchSpanProcessor.Export.Error`, to be consistent with `BatchLogProcessor.Export.Error`, as the error occurs while processing export message.